internal/credspropagation/common.go

```diff
@@ -53,11 +53,11 @@ func applyCCMConfigs(ctx context.Context, kubeconfSecret *corev1.Secret, objects
 	return nil
 }
 
-func makeSecret(name, namespace string, data map[string][]byte) *corev1.Secret {
+func makeSecret(name string, data map[string][]byte) *corev1.Secret {
```

Comment:
https://github.com/Mirantis/hmc/actions/runs/12067350380/job/33650115071

This is the reason for this change. I am not sure how it passed lint for other functions which I have updated too. Basically it would always get `metav1.NamespaceSystem` which would always evaluate to `kube-system`.

I think when we get to that point of changing the namespace, we can parameterize this.

Reply:
Well, shit...
Got it, then please adjust `makeConfigMap` as well, so both functions have the same signature. 
It would be good also to rename them to reflect that it will make secret in kube ns, but it's up to you really.

Reply:
Have updated makeConfigMap as well. But haven't renamed it to leave it flexible for future. (if/when we add a namespace param)